### DISCUSSION TITLE
Converting TRMM into Binary

### DISCUSSION THREAD

**Discussion by vibinj** [NASA-LIS/LISF#1236] [Q&A] [answered]
Hi, 
I am trying to convert TRMM precipitation data into binary. I have codes batch_convert.sh, input.dat, and myselect_vNsds.c. How to run the myselect_vNsds.c code and create executable. I have tried this "gcc myselect_vNsds.c -o myselect_vNsds.exe" , but unfortunately did not work. Please help.

Vibin Jose

**Comment by dmocko** [comment, marked as answer]
Hello,

Which TRMM collection are you trying to use?

We are not the TRMM data producers, so we are not able to help with TRMM data conversion.

Can you instead use the IMERG datasets, which LIS reads in their native format?

**Comment by vibinj** [comment]
Thank you,
I am using 3B42.20070201.00.7A.HDF .
